.github/workflows/ci.yml

```diff
@@ -282,19 +283,19 @@ jobs:
     if: github.event_name != 'pull_request'
     steps:
       - name: Checkout current branch
-        uses: actions/checkout@v3.0.2
+        uses: actions/checkout@v4.1.1
         with:
           fetch-depth: 0
       - name: Setup Java
-        uses: actions/setup-java@v2.5.0
+        uses: actions/setup-java@v4.2.1
         with:
           distribution: temurin
-          java-version: 8
```

Comment:
That changes triggers a compile error when trying to compile with Java 11:
```
[error] Uncaught exception when running tests: java.lang.UnsupportedClassVersionError: zio/internal/MutableQueueFieldsPadding has been compiled by a more recent version of the Java Runtime (class file version 61.0), this version of the Java Runtime only recognizes class file versions up to 55.0
```
I think we have to compile with the lowest supported Java version. Can you fix it?

Reply:
Opened a PR https://github.com/zio/zio/pull/8780

Reply:
My bad! Thanks 🙂